tests/transports-integrations/tests/utils/config_loader.py

```
@@ -0,0 +1,309 @@
+        endpoint = bifrost_config["endpoints"].get(integration, "")
+
+        if not endpoint:
+            raise ValueError(f"No endpoint configured for integration: {integration}")
+
+        return f"{base_url.rstrip('/')}{endpoint}"
+
+    def get_bifrost_config(self) -> BifrostConfig:
+        """Get Bifrost configuration"""
+        bifrost_data = self._config["bifrost"]
+        return BifrostConfig(
+            base_url=bifrost_data["base_url"], endpoints=bifrost_data["endpoints"]
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)